e2e-tests/stream-mode-tests.sh

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env bash
+
+#######################################
+# Function that prints messages
+# Arguments:
+#   anything that needs printing.
+#######################################
+function print_message() {
+  local print_prefix="E2E: STREAM MODE TEST:"
+  echo "${print_prefix} $*"
+}
+
+#######################################
+# Call common-mode-tests to setup 
+# the testing env
+#  
+#######################################
+source ./e2e-tests/common-mode-tests.sh
+setup
+
+print_message "OPENMRS AND FHIR SERVER STARTED SUCCESSFULLY"
+
+test_dir=$(mktemp -d -t analytics_tests__XXXXXX)
+
+print_message "PARQUET FILES WILL BE WRITTEN INTO $test_dir DIRECTORY"
+
+print_message "BUILDING THE ANALYTICS  PIPELINE"
+
+timeout 1m mvn compile exec:java -pl streaming-binlog \
+   -Dexec.args="--openmrsUserName=admin --openmrsPassword=Admin123 \
+   --openmrsServerUrl=http://localhost:8099/openmrs \
+   --openmrsfhirBaseEndpoint=/ws/fhir2/R4 \
+   --outputParquetPath='${test_dir}' --fhirSinkPath=http://localhost:8098/fhir --sinkUserName=hapi --sinkPassword=hai "
+
+
+print_message "SEARCH INDEX REBUILD"
+
+curl -i -u admin:Admin123 -X POST http://localhost:8099/openmrs/ws/rest/v1/searchindexupdate
+
+print_message "CREATE A PATIENT IN OPENMRS TO BE STREAMED"
+
+curl -i -u admin:Admin123 -H "Content-type: application/json" -X POST http://localhost:8099/openmrs/ws/fhir2/R4/Patient -d '{ 
+    "resourceType": "Patient",
```

Comment:
@gitcliff , i think you can place this resource somewhere in a json file and reference it by `curl --data "@/path/to/filename"`